### PR TITLE
add live validate

### PR DESCRIPTION
**Pull request by Nanhtu187**

### CODE REVIEW

File: lib/utils/helpers/getComponent.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ export const GetTailwindInputComponent = (
   key: string,
   formData: any,
   handleInputChange: (event: React.ChangeEvent<HTMLInputElement>) => void,
+  onBlur: (event: React.FocusEvent<HTMLInputElement>) => void,
   description?: string,
   errors?: string[],
 ): JSX.Element => {
```
Comment (tu-hm1) [inline lib/utils/helpers/getComponent.tsx:13]: don't need to add this type

File: lib/utils/helpers/getComponent.tsx [hunk 2]
```diff
@@ -18,6 +19,7 @@ export const GetTailwindInputComponent = (
         value={formData}
         type={"number"}
         onChange={handleInputChange}
+        onBlur={onBlur}
         description={description}
         errors={errors}
       />
```
Comment (tu-hm1) [inline lib/utils/helpers/getComponent.tsx:25]: for this condition, use switch case is better and readability more.

File: lib/utils/src/types.ts [hunk 3]
```diff
@@ -43,6 +43,7 @@ export interface BaseInputProps {
   label: string;
   value: any;
```
Comment (tu-hm1) [inline lib/utils/src/types.ts:44]: avoid any, use generic for this one.

File: lib/components/tailwind/input/input_field/index.tsx [hunk 4]
```diff
@@ -9,7 +9,11 @@ export interface InputProps extends BaseInputProps {
 
 export const InputField = (props: InputProps) => {
```
Comment (tu-hm1) [inline lib/components/tailwind/input/input_field/index.tsx:10]:
```suggestion
export const InputField = ({label, onBlur, ...} : InputProps) => {
```

File: lib/utils/helpers/getComponent.tsx (hunk 1 above)
Comment (Nanhtu187) [reply lib/utils/helpers/getComponent.tsx]: done

File: lib/utils/helpers/getComponent.tsx (hunk 2 above)
Comment (Nanhtu187) [reply lib/utils/helpers/getComponent.tsx]: done

File: lib/utils/src/types.ts (hunk 3 above)
Comment (Nanhtu187) [reply lib/utils/src/types.ts]: done

File: lib/components/tailwind/input/input_field/index.tsx (hunk 4 above)
Comment (Nanhtu187) [reply lib/components/tailwind/input/input_field/index.tsx]: done

**Pull request merged**